frontend/src/features/ratings/MarkAllRatings.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+import { Button, Box } from '@material-ui/core';
+import { Visibility, VisibilityOff } from '@material-ui/icons';
+import { FilterSection } from 'src/components';
+import { UsersService } from 'src/services/openapi';
+import { RatingsContext } from 'src/features/videos/PublicStatusAction';
+
+function MarkAllRatings() {
+  const { onChange: onRatingChange } = useContext(RatingsContext);
+
+  const updateRatings = async (isPublic: boolean) => {
+    await UsersService.usersMeContributorRatingsAllPartialUpdate({
+      requestBody: { is_public: isPublic },
+    });
+    if (onRatingChange) {
+      onRatingChange();
+    }
+  };
+
+  return (
+    <FilterSection title="Update visibility">
```

Comment:
A bit confusing here to use FilterSection for something that is not at all a filter. Maybe leave a comment here to alleviate the possible confusion or we could rename to component FilterSection to make explicit that is can be used for other reasons.

Reply:
I see your point. Any suggestion about the name?
Should I rename `FilterSection` to `OptionsSection` and `FiltersButton` to `OptionsButton`, or is it too generic?